pkg/machine/ignition.go

```diff
@@ -310,6 +312,7 @@ machine_enabled=true
 	delegateConf := `[Service]
 Delegate=memory pids cpu io
 `
+	subUID := `%s:100000:65336`
```

Comment:
```suggestion
	subUID := `%s:100000:65536`
```

Reply:
Shouldn't this be 65536? Mainly because this is used in the following test:
https://github.com/containers/podman/blob/ac5bbf165cc00bf1804cc05addb8c6230630d19b/pkg/rootless/rootless_test.go#L35

Reply:
Shouldn't this be 65536? Mainly because this is used in the following test:
https://github.com/containers/podman/blob/ac5bbf165cc00bf1804cc05addb8c6230630d19b/pkg/rootless/rootless_test.go#L35

Reply:
yup just a typo